spec/schemas/_common.yaml

```diff
@@ -20,137 +22,164 @@ components:
       required:
         - acknowledged
     DocStatus:
-      description: |-
-        The item level REST category class codes during indexing.
+      description: The item level REST category class codes during indexing.
       type: object
       properties:
         1xx:
           type: integer
           format: int32
+          description: The number of informational responses.
         2xx:
           type: integer
           format: int32
+          description: The number of successful responses.
         3xx:
           type: integer
           format: int32
+          description: The number of redirection responses.
         4xx:
           type: integer
           format: int32
+          description: The number of client error responses.
         5xx:
           type: integer
           format: int32
+          description: The number of server error responses.
```

Comment:
@nhtruong: This would likely modify line 25 then. This description defines what `5xx` means.